src/components/calendar/datepickerstyles.css

```diff
@@ -0,0 +1,27 @@
+.react-datepicker__day:hover {
+  background-color: inherit;
+  color: inherit;
+}
+
+.react-datepicker__day--selected {
+  background-color: transparent;
+}
+
+.react-datepicker__header {
+  background-color: #282b2d;
+  border: none;
+}
+
+.react-datepicker__current-month,
+.react-datepicker__day-name {
+  color: white;
+}
+
+.react-datepicker__day--outside-month {
+  visibility: hidden;
+}
```

Comment:
What are these selectors? Don't they provide a better way to access and edit those styles?

Reply:
these selectors are the onlky way I found to change certain styles in the calendar. will look for other ways.

Reply:
```
// https://reactdatepicker.com/#example-custom-calendar-class-name
// from the Sources tab in devtools: example.scss
118. .rasta-stripes {
119.   .react-datepicker__week:nth-child(3n + 1) {
120.     background-color: #215005;
121.   }
122.   .react-datepicker__week:nth-child(3n + 2) {
123.     background-color: #eea429;
124.   }
125.   .react-datepicker__week:nth-child(3n + 3) {
126.     background-color: #a82a15;
127.   }
128. }
```
Shit... 🌚